### DISCUSSION TITLE
/cmi compass not work when target is player

### DISCUSSION THREAD

**Discussion by kassadGLA** [Zrips/CMI#1949] [Enhancement]
Hello.
If you specify a player as a target, the compass will aim at its current coordinates, which will not be updated when the target player moves. For example, if I specify a player as a target when he was at coordinates `1, 1, 1`, then the compass would always point to coordinates `1, 1, 1`.

Best regards

---

**Cmi Version:** 
```
--------------------------------------------------
CMI: 8.6.5.1
Server: Paper(95) 1.14.2-R0.1-SNAPSHOT
Vault: 1.6.11(+)
--------------------------------------------------
```

**Server Type:** 
`Paperspigot`

**Server Version:** 
`Paper version git-Paper-95 (MC: 1.14.2) (Implementing API version 1.14.2-R0.1-SNAPSHOT)`

**Comment by Zrips** [comment]
Well this is vanilla behavior, you can't really aim at moving target with a compass, its only points to a static location. So to have your desired feature  (not a bug) we will need to add scheduler which changes compass target location to target player current position constantly.
